scripts/enrichment.py

```diff
@@ -1,26 +1,39 @@
-import requests
-from bs4 import BeautifulSoup, SoupStrainer
-from fetch_spotify import spotify_get
-from fetch_imdb import *
-from fetch_goodreads import *
-
-APIs = ["www.goodreads.com", "www.imdb.com", "open.spotify.com"]
-
-
-def enrich_test(url):
-    site_name = url.split("/")[2]
-
-    enrich_data = {}
-
-    if site_name in APIs:
-
-        if site_name == "www.goodreads.com":
-            enrich_data = fetch_goodreads(url)
-
-        elif site_name == "www.imdb.com":
-            enrich_data = fetch_imdb(url)
-
-        elif site_name == "open.spotify.com":
-            enrich_data = spotify_get(url)
-
-    return enrich_data
+import requests
+from bs4 import BeautifulSoup, SoupStrainer
+from fetch_spotify import spotify_get
+from fetch_imdb import *
+from fetch_goodreads import *
+from fetch_artsy import *
+from typing import Dict
+from constants import APIs
+
+def get_url(url: str) -> str:

```

Comment:
We created a test for this function using pytest. The function was passing all the cases. Here's the test file:
[test.zip](https://github.com/analogue-app/mercury/files/4601531/test.zip)

Reply:
🔥awesome! Let's figure out a way to integrate this into the app so we can all run these tests. Or we can integrate into the workflow and have them run everytime we open a PR to make sure all tests are passing. @hxghhhh what's best practice for this in python/flask?

Reply:
Great, but this is the bare minimum. We'll develop it over time as we learn more about pytest. 🙂